src/repl/scala/tools/nsc/interpreter/ReplProps.scala

```diff
@@ -32,6 +32,12 @@ class ReplProps {
   val powerInitCode   = Prop[JFile]("scala.repl.power.initcode")
   val powerBanner     = Prop[JFile]("scala.repl.power.banner")
 
+  val prompt = Prop[String]("scala.shell.prompt")
```

Comment:
How is that related to `"shell.prompt"` that we already have here? https://github.com/scala/scala/blob/2.11.x/src/compiler/scala/tools/nsc/Properties.scala#L16